CHANGELOG.md

```diff
@@ -91,6 +91,7 @@ For older changes see the [archived Singularity change log](https://github.com/a
   binding fakeroot into container during apptainer startup for --fakeroot
   with fakeroot command.
 - Fix memory usage calculation during apptainer compilation on RaspberryPi.
+- Improve message when running encrypted sif without a key material option.
```

Comment:
Since unprivileged SIF encryption hasn't been released yet, there's no need for a CHANGELOG entry.

Reply:
thanks, removed the entry in the CHANGELOG.md